README.md

```diff
@@ -104,11 +104,11 @@ After you're finished please remove all the comments and instructions!
 
 <!-- LIVE DEMO -->
 
-## 🚀 Live Demo <a name="live-demo">https://www.loom.com/share/8276eadf384d4fd29e7a01214553342a</a>
+## 🚀 Live Demo <a name="live-demo"></a>
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://www.loom.com/share8276eadf384d4fd29e7a01214553342a)
```

Comment:
- [ ] This issue is not yet resolved :x: 

- As requested by the previous reviewer, please kindly deploy your app and add the live demo link to the readme to make the readme more professional. Here is meant for the live demo link, then you can add another link to denote the video presentation link